rangefilter/tests.py

```diff
@@ -274,3 +275,18 @@ def test_datefilter_filtered_with_one_params(self):
         choice = select_by(filterspec.choices(changelist))
         self.assertEqual(choice['query_string'], '?')
         self.assertEqual(choice['system_name'], 'created-at')
+
+
+class StaticOrAdminStaticTestCase(TestCase):
+
+    def test_renders_static_path_to_asset(self):
```

Comment:
Look at the [cool test](https://github.com/django/django/blob/f0f383b6350c7b2f64bb2a6491a395abdd3313d6/tests/admin_views/tests.py#L203) in the Django code. This test does not require `TEMPLATES` in settings.

Reply:
I like the way the template tag is tested in the code you pointed out. I think it is more precise. Thanks for the suggestion! I'll try to follow that guideline in the PR.